components/UserInformation/UserInformation.spec.jsx

```diff
@@ -0,0 +1,62 @@
+// Mount import
+import { mount } from '@cypress/react';
+
+// Component import
+import UserInformation from './UserInformation';
+
+// As we don't pass any additional props, we can extract the mount call.
+beforeEach(() => {
+  mount(<UserInformation />);
+});
+
+/**
+ * User information component tests.
+ */
+describe('User information', () => {
+  it('Shows user information component with default props', () => {
+    cy.get('[class*="container"]')
+      .should('exist')
+      .should('be.visible');
+  });
+
+  it('Initially shows default "John Doe" username greeting', () => {
+    cy.get('[class*="name"]')
+      .should('include.text', 'Hi John Doe!');
+  });
+
+  it('Shows information for the user input', () => {
```

Comment:
I don't think this test is really necessary.

Reply:
Maybe not. I just wanted to write tests that will check that everything the component renders is rendered properly. You could also argue that all those `it` tests could be defined inside the first test

Reply:
I agree. On almost any other component with more elements this would become painful very quickly

Reply:
Yes.